metrics.go

```diff
@@ -53,7 +53,7 @@ var (
 	sitePowerPhotovoltaicsGauge = promauto.NewGauge(prometheus.GaugeOpts{
 		Namespace: namespace,
 		Name:      "site_power_photovoltaic",
-		Help:      "Site power supplied to or provided from the accumulator(s) in Watt",
+		Help:      "Site power from photovoltaic",
```

Comment:
Please leave the unit in the help
```suggestion
		Help:      "Site power from photovoltaic in Watt",
```